pycbc/inference/models/base.py

```diff
@@ -738,7 +738,8 @@ def _init_args_from_config(cls, cp):
         args['sampling_transforms'] = sampling_transforms
         # get any other keyword arguments provided
         args.update(cls.extra_args_from_config(cp, section,
-                                               skip_args=['name']))
+                                               skip_args=['name'],
+                                               dtypes={'lower_frequency_cutoff':float}))
```

Comment:
Ok, I know it was easiest to just stick this here, but there's a couple of issues with this. First, it doesn't make a lot of sense to have something about lower frequency cutoff in BaseModel, since BaseModel may be used by models that don't involve data. I realize that having that there doesn't affect the running of the code, but the location is obscured. It'd be tough for someone to figure out where in the code the lower frequency cutoff was being set. Second, @soumide1102  is going to adding more functionality after this patch is merged to use different cutoff frequencies for different ifos, so will need some place to write extra lines of code.

So, I suggest removing that, and instead adding a an `_init_args_from_config` function to the `GaussianNoise` model, similar to what's done [here](https://github.com/gwastro/pycbc/blob/c989c1e5a73745e55708feec0c70299248f1f802/pycbc/inference/models/base_data.py#L168-L174). After you call `super(GaussianNoise, self)._init_args_from_config(cp)`, add a couple of lines to read `low-frequency-cutoff` from the `cp` file and add it to the `args`. Later, @soumide1102 can add the extra lines there to parse the low frequency cutoff option for different ifos.

Reply:
Ok